domains.txt

```diff
@@ -325,6 +363,7 @@ initi8recruitment.com
 initialize-it.com
 insideconsultancy.com
 insource-select.co.uk
+inspiredtalent.co.uk
```

Comment:
Inspired Talent was established in 2014 and has quickly gained traction across several market sectors. Our unique approach to the way we provide a quality recruitment service, has set us apart from other recruiters and helped establish our brand within a very saturated industry.